### PR TITLE
Add a wLog syslog appender

### PR DESCRIPTION
**Pull request by hardening**

### CODE REVIEW

**Comment by hardening** [comment]
Updated to last changes on wLog

File: config.h.in [hunk 1]
```diff
@@ -32,6 +32,7 @@
 #cmakedefine HAVE_TM_GMTOFF
 #cmakedefine HAVE_AIO_H
 #cmakedefine HAVE_POLL_H
+#cmakedefine HAVE_SYSLOG_H
```
Comment (akallabeth) [inline config.h.in]: @hardening Probably better to define this via compiler argument. Otherwise you'll have to include ```config.h``` in ```wlog.h``` too.

**Comment by akallabeth** [comment]
except for the definition remark +1

**Comment by hardening** [comment]
@akallabeth I have treated your ramark

**Comment by bmiklautz** [comment]
Same remark as @akallabeth I think we shouldn't have config.h in an exposed header.
Otherwise it looks good. Nice work!
+1

**Comment by hardening** [comment]
Ok so no more config.h include in public headers.

**Pull request merged**